### PR TITLE
[CoW Subsidy] - connect real hook to other stuff

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Connects @nenadV91 's work on the vCoW total balance to the subsidy stuff
Part of #2549 

## What's missing:
- [ ] actually change fee discount in the fee shown

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
Seems to be working fine
Screen Shot 2022-03-22 at 15 46 18

Although can't really tell the discount when looking at gchain
Screen Shot 2022-03-22 at 15 53 05

File: src/custom/hooks/useCowBalanceAndSubsidy.ts [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
-import { CurrencyAmount } from '@uniswap/sdk-core'
 import { BigNumber } from 'bignumber.js'
-import { parseUnits } from '@ethersproject/units'
 
 import { getDiscountFromBalance } from 'components/CowSubsidyModal/utils'
-import { SupportedChainId } from 'constants/chains'
-import { V_COW } from 'constants/tokens'
-import { useActiveWeb3React } from '.'
+import { useVCowData } from 'state/claim/hooks'
+import { COW_SUBSIDY_DATA } from '../components/CowSubsidyModal/constants'
```
[the remark this answers is not in the document]
Comment (W3stside) [reply src/custom/hooks/useCowBalanceAndSubsidy.ts:6]: You and PATHS. Jk yeah yeah

**Comment by elena-zh** [comment]
Hey @W3stside , I can see that the subsidy amount is changes according to the available balance in the wallet, however, I'm concerned that it does not affect a trade: there is not discounts in the tooltips for trades. Will it be implemented?

File: src/custom/hooks/useCowBalanceAndSubsidy.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (W3stside) [reply src/custom/hooks/useCowBalanceAndSubsidy.ts:6]:
```suggestion
import { COW_SUBSIDY_DATA } from 'components/CowSubsidyModal/constants'
```

**Comment by W3stside** [comment]
> Hey @W3stside , I can see that the subsidy amount is changes according to the available balance in the wallet, however, I'm concerned that it does not affect a trade: there is not discounts in the tooltips for trades. Will it be implemented? 

not yet! next PR

**Comment by W3stside** [comment]
actually i take this abck there's nothing for FE to do the rest is handled in the backend, they handle the fee subsidisation

**Pull request merged**